client/my-sites/domains/domain-management/domain-overview-pane/index.tsx

```diff
@@ -140,20 +143,43 @@ const DomainOverviewPane = ( {
 				preview: enabled ? selectedDomainPreview : null,
 			};
 		} );
-	}, [ __, selectedDomain, selectedFeature, selectedDomainPreview ] );
+	}, [ selectedFeature, selectedDomainPreview, inSiteContext, selectedDomain, siteSlug ] );
+
+	const isMigrationPending = getMigrationStatus( site ) === 'pending';
+	const faviconFallback = isMigrationPending ? 'migration' : 'first-grapheme';
 
 	return (
-		<ItemView
-			itemData={ itemData }
-			closeItemView={ () => {
-				inSiteContext ? page.show( '/sites' ) : page.show( paths.domainManagementRoot() );
-			} }
-			features={ features }
-			enforceTabsView
-			itemViewHeaderExtraProps={ {
-				headerButtons: PreviewPaneHeaderButtons,
-			} }
-		/>
+		<>
+			{ inSiteContext && (
+				<div className="domain-overview__breadcrumb">
+					<Breadcrumb
+						items={ [
+							{
+								label: site.name || selectedDomain,
+								href: `/overview/${ siteSlug }`,
+								icon: <SiteFavicon blogId={ site.ID } size={ 24 } fallback={ faviconFallback } />,
+							},
+							{
+								label: selectedDomain,
+							},
```

Comment:
Can we increase the font size by another px? It's currently 13px, but in the design it's 14px

Reply:
Fixed, thanks!